napalm/eos/utils/cli_syntax.py

```diff
@@ -0,0 +1,357 @@
+        "show user-account": "show users accounts",
+        "show vlan summary": "show vlan brief count",
+        "snmp trap link-status": "snmp trap link-change",
+        "snmp-server source-interface": "snmp-server local-interface",
+        "spanning-tree bridge assurance": "spanning-tree transmit active",
+        "spanning-tree loopguard default": "spanning-tree guard loop default",
+        "spanning-tree portfast bpdufilter default": "spanning-tree edge-port bpdufilter default",
+        "spanning-tree portfast bpduguard default": "spanning-tree edge-port bpduguard default",
+        "spanning-tree transmit hold-count": "spanning-tree bpdu tx hold-count",
+        "spanning-tree vlan": "spanning-tree vlan-id",
+        "statistics per-entry": "counters per-entry",
+        "switchport backup interface": "switchport backup-link",
+        "switchport port-security maximum": "switchport port-security mac-address maximum",
+        "switchport vlan mapping": "switchport vlan translation",
+        "timers basic": "timers",
+        "timers lsa arrival": "timers lsa rx min interval",
+        "timers throttle lsa all": "timers lsa tx delay initial",
+        "timers throttle spf": "timers spf delay initial",
+        "username sshkey": "username ssh-key",
+        "vlan internal allocation policy": "vlan internal order",
+        "vrf definition": "vrf instance",
+        "vrf forwarding": "vrf",
+        "vrrp authentication": "vrrp peer authentication",
+        "vrrp delay reload": "vrrp timers delay reload",
+        "vrrp description": "vrrp session description",
+        "vrrp ip": "vrrp ipv4",
+        "vrrp ip secondary": "vrrp ipv4 secondary",
+        "vrrp priority": "vrrp priority-level",
+        "vrrp shutdown": "vrrp disabled",
+        "vrrp timers advertise": "vrrp advertisement interval",
+        "vrrp track": "vrrp tracked-object",
+    },
+}
+
+
+class CliConverter:
+    """
+    ClI converter class
+    """
+
+    def __init__(self, syntax):
+        """
+        Object creation
+        :param syntax: syntax dict
+        """
+        self.syntax = syntax
+
+    def convert(self, command, version):
+        """
+        Convert command from version 1 to specified version
+        :param command: str: command
+        :param version: int: version number
+        :return: str: command
+        """
+        if version not in self.syntax:
+            return command
+
+        for c in self.syntax[version]:
+            if command.startswith(c):
+                return self.syntax[version][c] + command[len(c) :]
+
+        return command
+
+
+CONVERTER = CliConverter(CLI_SYNTAX)
+
+
+def cli_convert(command, version):
+    """
+    Convert command from CLI version 1 to one from CLI of specified version
+    :param command: str: CLI command
+    :param version: int: EOS CLI version number
+    :return: str: command
+    """
+    return CONVERTER.convert(command, version)
```

Comment:
This feels a bit redundant. I'd probably just do:

```python
def cli_convert(command, version):
        if version not in CLI_SYNTAX:
            return command

        for c in CLI_SYNTAX[version]:
            if command.startswith(c):
                return CLI_SYNTAX[version][c] + command[len(c) :]

        return command
```
And get rid of the `CliConverter` class.

Not saying to do so, just thinking out loud. :-)